ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenRunner.java

```diff
@@ -318,11 +319,18 @@ public Void visitLambdaExpression(
     }
 
     private void addRequiredColumn(final ColumnName columnName) {
+      final String informationAboutPseudoColumns =
```

Comment:
nit: let's only compute this if we need to the throw the error message. You can define a helper method `fieldNotFoundErrorMessage(columnName)` to return the appropriate error message string. Besides being more efficient, this will also help with readability.